src/services/dynamic-import.ts

```diff
@@ -6,4 +6,6 @@
  * subject to an additional IP rights grant found at
  * http://polymer.github.io/PATENTS.txt
  */
+
+// @ts-ignore
```

Comment:
Please put the error you are ignoring in the comment.

Reply:
Is this satisfactory? aa58811

Reply:
looks like that should do it..